ppr-ui/src/views/mhrInformation/MhrTransportPermit.vue

```diff
@@ -164,41 +164,42 @@
         id="location-change-type-section"
         class="mt-7"
       >
-        <FormCard
-          label="Location Change Type"
-          :showErrors="false"
-          :class="{'border-error-left': false}"
-        >
-          <template #formSlot>
-            <v-select />
-          </template>
-        </FormCard>
+        <LocationChange
+          :content="{ sideLabel: 'Location Change Type' }"
+          @updateLocationType="handleLocationTypeUpdate($event)"
+        />
       </section>
     </template>
   </div>
 </template>
 
 <script setup lang="ts">
-import { Ref, ref } from 'vue'
-import { FormCard, SimpleHelpToggle } from '@/components/common'
-import DocumentId from '@/components/common/DocumentId.vue'
-import { useStore } from '@/store/store'
-import { useTransportPermits } from '@/composables'
+import { DocumentId, SimpleHelpToggle } from "@/components/common"
+import { LocationChange } from "@/components/mhrTransfers"
+import { useTransportPermits } from "@/composables/mhrInformation"
+import { useStore } from "@/store/store"
+import { ref } from "vue"
 
-// Props
-defineProps<{
-  disable?: boolean
-}>()
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const props = defineProps({
+  disable: {
+    type: Boolean,
+    default: false
+  }
+})
```

Comment:
I believe you can defineProps without assigning to a variable which would also wouldn't require the lint disable comment. Same as above in the Location Change component.

Reply:
Initially the test was failing because it couldn't access the props... I just tried again and it seems to be passing now. I'll push new commit.